Switch used-by to mdakits page (#323)
MDAnalysis/MDAnalysis.github.io: pages/mdakits.md

---
layout: page
title: MDAKits and other tools using MDAnalysis
---

MDAnalysis is developed with extensibility in mind, allowing
scientists to create various tools with the components
that the library provides. To help developers in this process,
and to advertise these wonderful tools to the MDAnalysis user community,

> we have created an ecosystem of [MDAnalysis Toolkits](https://mdakits.mdanalysis.org/about.html).

Reviewer: ```suggestion
we have created an ecosystem of [MDAnalysis Toolkits, or "MDAKits"](https://mdakits.mdanalysis.org/about.html).
```

> Please look at the [registry of toolkits](https://mdakits.mdanalysis.org/mdakits.html)

Reviewer 2: ```suggestion
Please look at the [registry of MDAKits](https://mdakits.mdanalysis.org/mdakits.html)
```

> for a list of toolkits that [meet the minimum MDAKit requirements](insert link here).

Author: Hold for Oliver's PR on the mdakits registry.

These are regularly tested against the latest and development versions of

> MDAnalysis, so if you see a green badge it should be ok to use!

Reviewer: ```suggestion
MDAnalysis, so if you see a green badge, we expect that it should be good to use!
```

"Ok" felt a little unenthusiastic -- minor comment, but swap for "good"?

Author: 🤔 We probably should be even clearer that it's not on us but on the MDAKit authors, I might try something else.

Author: Updated

*If you have your own MDAKit and you would like to add it to the
registry, please look at the [contribution instructions](https://mdakits.mdanalysis.org/add.html)
or share it with on our [mailing list]({{ site.mailinglists.discussion.url }})!*

## Other (non-MDAKit registered) tools using MDAnalysis

> Below is a list of non-MDAKit registered projects which we know to use MDAnalysis:

Reviewer: ```suggestion
Below we list projects that use MDAnalysis and are not registered as MDAKits:
```

Stylistic suggestion
